thesis-gui/src/components/my-quizzes/MyQuizzes.tsx

```diff
@@ -1,77 +1,98 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import {
     IonButton,
-    IonContent,
+    IonContent, IonIcon,
     IonItem,
     IonLabel,
     IonList,
-    IonPage
+    IonPage, IonText, useIonToast
 } from "@ionic/react";
+import {keyOutline} from "ionicons/icons";
 import {PageHeader} from "../common/PageHeader";
+import {quizzesService} from "../../services/quizzes-service";
+import {QuizzesList} from "../../types/quizzes-list";
 
 const PAGE_NAME = "My Quizzes";
 
-const QUIZZES = [
-    {
-        name: "Quiz 1",
-        questionsNumber: 5
-    },
-    {
-        name: "Quiz 2",
-        questionsNumber: 27
-    },
-    {
-        name: "Quiz 3",
-        questionsNumber: 8
-    },
-    {
-        name: "Quiz 4",
-        questionsNumber: 13
-    }
-];
-
 const QuizzesListItem: React.FC<{
     name: string,
-    questionsNumber: number
-}> = ({ name, questionsNumber }) => {
+    questionsNumber: number,
+    _key: string
+}> = ({ name, questionsNumber, _key }) => {
+    const [present] = useIonToast();
+
+    const presentToast = () => {
+        present({
+            message: `"${name}" quiz key was copied to clipboard!`,
+            duration: 1500,
+            position: 'bottom'
+        });
+    };
+
+    const copyKeyToClipboard = () => {
+        navigator.clipboard.writeText(_key);
+        presentToast();
+    }
+
     return (
-        <IonItem button detail={true}>
+        <IonItem>
             <IonLabel>
                 <h2>{ name }</h2>
                 <p>{ `${questionsNumber} question(s)` }</p>
             </IonLabel>
+            <IonIcon
+                className="btn"
```

Comment:
Does this class name exist? I tried removing it and nothing seems to change.

Reply:
I don't remember why I added it here to be honest 😆 Probably I wanted to create `btn` class in `.css` file but I don't know 🤷‍♀️ 
I will change it the way I proposed you in Quiz Creator PR